stdlib/profiler/profiler.flux

```diff
@@ -0,0 +1,3 @@
+package profiler
+
+option enabledProfilers = [""]
```

Comment:
Why does this need to be a list of empty string? is it so that the type system knows is a list of strings?

Reply:
Yes